### ISSUE TITLE
preservation object handler service method for create (blocked by #65)

### ISSUE THREAD

**Issue by jmartin-sul** [sul-dlss/preservation_core_catalog#113]
something that does the right default stuff for creating a new catalog entry:
* create the preserved_object
* create the online preservation_copy
* give it a default preservation_policy
* whatever else

**Comment by ndushay** [comment]
duplicate of #131;  closing.
